src/Two/GoogleProvider.php

```diff
@@ -76,15 +76,18 @@ protected function getUserByToken($token)
      */
     protected function mapUserToObject(array $user)
     {
-        $avatarUrl = Arr::get($user, 'picture');
+        // Deprecated: Fields added to keep backwards compatibility in 4.0. These will be removed in 5.0
+        $user['id'] = Arr::get($user, 'sub');
+        $user['verified_email'] = Arr::get($user, 'email_verified');
+        $user['link'] = Arr::get($user, 'profile');
 
         return (new User)->setRaw($user)->map([
-            'id' => $user['id'],
+            'id' => Arr::get($user, 'sub'),
             'nickname' => Arr::get($user, 'nickname'),
             'name' => Arr::get($user, 'name'),
             'email' => Arr::get($user, 'email'),
-            'avatar' => $avatarUrl,
-            'avatar_original' => preg_replace('/\?sz=([0-9]+)/', '', $avatarUrl),
+            'avatar' => $avatarUrl = Arr::get($user, 'picture'),
+            'original_avatar' => $avatarUrl,
```

Comment:
This is a backwards compat break. Changing the key avatar_original to original_avatar

Reply:
Unfortunately Laravel packages do not follow semantic versioning so things can break at any time.

Reply:
All packages by Laravel except Lumen and the framework itself follow SemVer. This was a mistake. I'll correct it.